src/storage-types/src/parameters.rs

```diff
@@ -111,6 +112,7 @@ impl Default for StorageParameters {
             enable_dependency_read_hold_asserts: true,
             user_storage_managed_collections_batch_duration:
                 STORAGE_MANAGED_COLLECTIONS_BATCH_DURATION_DEFAULT,
+            dyncfg_updates: None,
```

Comment:
not sure I follow this! the comment at the top talks about the defaults matching the LD defaults. we are able to pull the defaults for the dyncfgs here if that's needed?

Reply:
Oh does this just go away once we finish swapping all the other ones over to dyncfgs?

Reply:
@danhhz yes!